documentation/modules/deploying/con-securing-kafka-connect-api.adoc

```diff
@@ -0,0 +1,73 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deploy-kafka-connect-with-plugins.adoc
+
+[id='con-securing-kafka-connect-api-{context}']
+= Limiting access to the Kafka Connect API
+
+[role="_abstract"]
+To grant a user access to the Kafka Connect API, a Kafka cluster admin must provide them with the API URL. 
+This includes the hostname or IP address of the Kafka Connect cluster and the port number on which the API is running (by default, port 8083). 
+Additionally, the user must have authenticated access to the Kubernetes cluster and the Kafka Connect REST API endpoint in order to make a request.
```

Comment:
The subject is _limiting_ access but we're starting out with how to _give_ access, which is a little weird. 

I think we should start with the "Why?" (which is currently the 2nd paragraph), and then go on to the "How?" (which is like this paragraph, but stated slightly differently):

```suggestion
Anyone with authenticated access to the Kubernetes cluster and knowledge of the Kafka Connect REST API endpoint (URL including hostname/IP address and port number) can use the REST API.
```

Then the example shows how someone with such access is able to use their ability to create a connector to recover a secret which they'd not otherwise have access to.

Reply:
Thanks @tombentley. I've done the switch and changed to a new second paragraph as follows:

> The Kafka Connect REST API can be accessed by anyone who has authenticated access to the Kubernetes cluster and knows the endpoint URL, which includes the hostname/IP address and port number.